lib/src/polar_base.dart

```diff
@@ -537,6 +537,11 @@ class Polar {
     );
   }
 
+  /// Set local time of the device
+  Future<void> setLocalTime(String identifier) {
```

Comment:
There should be a `time` field here that gets passed to the native platforms and handled there